src/decoder/mod.rs

```diff
@@ -1071,28 +1151,67 @@ impl<R: Read + Seek> Decoder<R> {
         Ok(())
     }
 
+    pub fn chunk_info(&self, chunk_index: usize) -> Option<ChunkInfo> {
```

Comment:
I have now removed public and am now returning ChunkInfo from `read_tile()` and `read_tile_at()`. To keep consistency I have added in the same functionality for accessing strips - but I believe this will be consolidated anyway in a different PR?